content/blogposts/2019/gitlab-integrates-sourcegraph-code-navigation-and-code-intelligence.md

```diff
@@ -16,10 +16,12 @@ published: true
   alt="GitLab plus Sourcegraph logo"
 />
 
-<BlockquoteWithBorder
-  quote='Sourcegraph has the best find-definition, find-references, and intelligent code navigation capability on the planet - and they brought it to GitLab.'
-  author='Sid Sijbrandij, GitLab CEO'
-/>
+<section className="text-center">
```

Comment:
I made #5576 last week, so we can start to standardize our blockquotes even more. I agree, I think there's too many variations still, Tracey hasn't looked at these yet, so it's another spot for a small improvement.

For now I made a `center` prop for `Blockquote` in [65d56a0](https://github.com/sourcegraph/about/pull/5558/commits/65d56a04faa491ac81eb33cce72820202e98c93c). I think they ultimately should all be left or center aligned since the difference seems so small.

Reply:
Awesome! That looks a lot cleaner with the prop for the time being. Looking forward to standardizing in #5576!